jvcl/run/JvHidControllerClass.pas

```diff
@@ -755,6 +757,21 @@ function TJvHidPnPInfo.GetRegistryPropertyDWord(PnPHandle: HDEVINFO;
     RegDataType, PBYTE(@Result), SizeOf(Result), BytesReturned);
 end;
 
+function TJvHidPnPInfo.GetRegistryPropertyGuid(APnPHandle: HDEVINFO;
+  const ADevData: TSPDevInfoData; Prop: DWORD): TGuid;
+const
+  GUID_NULL: TGUID = '{00000000-0000-0000-0000-000000000000}';
```

Comment:
The RTL already has `EmptyGUID` defined, no need to redefine it. And if it is missing in older versions of Delphi, this constant my has well be defined in `JvJCLUtils` for instance